frontend/src/redux/store/store.ts

```diff
@@ -1,10 +1,26 @@
-import { createStore, applyMiddleware, compose } from 'redux';
+import { SDKReducers } from '@openshift/dynamic-plugin-sdk-utils';
+import { createStore, applyMiddleware, compose, combineReducers } from 'redux';
 import reduxThunk from 'redux-thunk';
 import appReducer from '~/redux/reducers/appReducer';
 
-// eslint-disable-next-line
-const composeEnhancers = (window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+const composeEnhancers =
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__?.({
+    name: 'Open Data Hub',
```

Comment:
Don't think it will really ever matter, but we have an ENV var for this that will change from ODH to RHOAI as needed...

Reply:
Fair enough. May as well use it if it's available.

Reply:
Updated to use the `ODH_PRODUCT_NAME` variable.